irc/websocket.go

```diff
@@ -0,0 +1,61 @@
+package irc
+
+import (
+	"github.com/gorilla/websocket"
+	"net/http"
+	"time"
+)
+
+var wsUpgrader = websocket.Upgrader{
+	ReadBufferSize:  2 * 1024,
+	WriteBufferSize: 2 * 1024,
+	// If a WS session contains sensitive information, and you choose to use
```

Comment:
TIL, thanks!

Reply:
Copy pasted from this https://github.com/oragono/oragono/blob/eae04e8c5157ac57a0cd7eeb11bc130c33bfb99a/irc/websocket.go#L17